tests/contrib/redis/test_redis.py

```diff
@@ -373,7 +373,7 @@ def test_patch_unpatch(self):
     @snapshot()
     def test_opentracing(self):
         """Ensure OpenTracing works with redis."""
-        writer = ddtrace.tracer.writer
+        writer = ddtrace.tracer._writer
```

Comment:
In all integrations but this case, we can avoid accessing the internal writer directly. To remove this dependency here would require addressing the `FIXME` highlighted below.